packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -168,6 +173,92 @@ function useProcessMedia({
     ]
   );
 
+  const trimExistingVideo = useCallback(
+    ({ resource: oldResource, start, end }) => {
+      const { src: url, mimeType } = oldResource;
+
+      const trimData = {
+        original: oldResource.id,
+        start,
+        end,
+      };
+
+      const onUploadStart = () => {
+        updateExistingElements({
+          oldResource: {
+            ...oldResource,
+            trimData,
+            isTrimming: true,
+          },
+        });
+      };
+
+      const onUploadError = () => {
+        updateExistingElements({
+          oldResource: { ...oldResource, isTrimming: false },
+        });
+      };
+
+      const onUploadSuccess = ({ resource }) => {
+        copyResourceData({ oldResource, resource });
+        updateOldMutedObject(oldResource.id, resource.id);
+        if ('video' === resource.type && !resource.local) {
+          if (!resource.posterId) {
+            uploadVideoPoster(resource.id, resource.src);
+          }
+          if (resource.isMuted === null) {
+            updateVideoIsMuted(resource.id, resource.src);
```

Comment:
A similar renaming of this seems required to avoid confusion.

Reply:
Oh wait, I'm probably mistaken about this one - it **does** have to do with muting.